bpf/lib/nodeport.h

```diff
@@ -2647,6 +2648,8 @@ int tail_nodeport_nat_egress_ipv4(struct __ctx_buff *ctx)
 	struct iphdr *ip4;
 	__s8 ext_err = 0;
 #ifdef TUNNEL_MODE
+	__u8 cluster_id __maybe_unused = (__u8)ctx_load_meta(ctx, CB_CLUSTER_ID_EGRESS);
+	__u32 src_identity = ctx_load_meta(ctx, CB_SRC_LABEL) ?: WORLD_IPV4_ID;
```

Comment:
I think we should do this selection in `nodeport_lb4()` already. Also note that we currently use `WORLD_IPV4_ID` when [creating](https://github.com/cilium/cilium/blob/fa0037638534635471c8e45583d78f87ad78b3e1/bpf/lib/nodeport.h#L2917) the nodeport CT entry, should probably stay in sync (even though I have no clue what it's used for ...).